From 577d87300906f961d13f76b44ba60fc40f0c637a Mon Sep 17 00:00:00 2001
From: William Jon McCann <william.jon.mccann@gmail.com>
Date: Fri, 14 Feb 2014 15:27:20 -0500
Subject: [PATCH] docs: fix mismatched para tags

newer gtkdoc is more sensitive to mismatched docbook tags.
This fixes the build in master.
---
 gst/audiofx/audiocheblimit.c | 4 ++--
 gst/udp/gstudpsrc.c          | 3 +--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/gst/audiofx/audiocheblimit.c b/gst/audiofx/audiocheblimit.c
index 549a9d2..e278886 100644
--- a/gst/audiofx/audiocheblimit.c
+++ b/gst/audiofx/audiocheblimit.c
@@ -46,12 +46,12 @@
  * be at most this value. A lower ripple value will allow a faster rolloff.
  *
  * As a special case, a Chebyshev type 1 filter with no ripple is a Butterworth filter.
- * </para>
+ *
  * <note><para>
  * Be warned that a too large number of poles can produce noise. The most poles are possible with
  * a cutoff frequency at a quarter of the sampling rate.
  * </para></note>
- * <para>
+ *
  * <refsect2>
  * <title>Example launch line</title>
  * |[
diff --git a/gst/udp/gstudpsrc.c b/gst/udp/gstudpsrc.c
index c2cbd92..7a24339 100644
--- a/gst/udp/gstudpsrc.c
+++ b/gst/udp/gstudpsrc.c
@@ -78,8 +78,7 @@
  * </itemizedlist>
  * The message is typically used to detect that no UDP arrives in the receiver
  * because it is blocked by a firewall.
- * </para>
- * <para>
+ *
  * A custom file descriptor can be configured with the
  * #GstUDPSrc:sockfd property. The socket will be closed when setting the
  * element to READY by default. This behaviour can be
-- 
1.8.5.3

